view/frontend/templates/subscription_freq_selector_product_page.phtml

```diff
@@ -21,151 +21,3 @@ if (!$block->isBoltProductPageForFrequencySelector()) { return;
-                            }
-                            options[checkboxAttribute].push(value);
-                            return;
-                        }
-                    }
-
-                    options[name] = value;
-                });
-                options['storeId'] = '<?= /* @noEscape */ $block->getStoreId(); ?>';
-                var formKey = $.mage.cookies.get('form_key');
-                if (!formKey && $('[name="form_key"]').length) {
-                    formKey = $('[name="form_key"]').val();
-                }
-                options['form_key'] = formKey;
-                var items = [];
-                <?php if ($block->isGrouped()): ?>
-                    <?php foreach ($block->getGroupedProductChildren() as $childProduct):?>
-                        var childProductQty = getGroupedProductChildQty(<?= /* @noEscape */ $childProduct->getId(); ?>);
-                        if (childProductQty > 0) {
-                            <?php if ($childProduct->isVirtual()): ?>
-                                var productType = 'digital';
-                            <?php else: ?>
-                                var productType = 'physical';
-                            <?php endif; ?>
-                            items.push({
-                                reference: '<?= /* @noEscape */ $childProduct->getId(); ?>',
-                                price: '<?= /* @noEscape */ $childProduct->getPrice() ?>',
-                                name: '<?= /* @noEscape */ addcslashes($childProduct->getName(), "'"); ?>',
-                                quantity: childProductQty,
-                                options: JSON.stringify(options),
-                                type: productType
-                            });
-                        }
-                    <?php endforeach; ?>
-                <?php else: ?>
-                    <?php if ($block->getProduct()->isVirtual()): ?>
-                        var productType = 'digital';
-                    <?php else: ?>
-                        var productType = 'physical';
-                    <?php endif; ?>
-                    items.push({
-                        reference: '<?= /* @noEscape */ $block->getProduct()->getId(); ?>',
-                        price: <?= /* @noEscape */ $block->getProduct()->getFinalPrice(); ?>,
-                        name: '<?= /* @noEscape */ addcslashes($block->getProduct()->getName(), "'"); ?>',
-                        quantity: getQty(),
-                        options: JSON.stringify(options),
-                        type: productType
-                    });
-                <?php endif; ?>
-
-                return items;
-        };
-
-        var cart;
-        var setupProductPage = function () {
-            cart = {
-                items: getItemsData()
-            };
-
-            // if connect.js is not loaded postpone until it is
-            whenDefined(window, 'BoltCheckout', function(){
```

Comment:
Without this, who will be calling our `configureProductFrequencySelector()` function to mount to our `bolt-freq-selector` div?